### ISSUE TITLE
New module request - meraki_client

### ISSUE THREAD

**Issue by kbreit** [ansible/ansible#51027]
##### SUMMARY
Meraki has increased support for detailed information retrieval from the API. They've also added support for provisioning clients, so there is now a good use for a `meraki_client` module.

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME
meraki

**Comment by ansibot** [comment]
Files identified in the description:
* [lib/ansible/module_utils/network/meraki/__init__.py](https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/network/meraki/__init__.py)

If these files are inaccurate, please update the `component name` section of the description or use the `!component` bot command.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
cc @dagwieers
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by kbreit** [comment]
This is waiting on #49015 for url encoding support in the URL.

**Comment by ansibot** [comment]
Files identified in the description:
None

If these files are incorrect, please update the `component name` section of the description or use the `!component` bot command.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by kbreit** [comment]
Marking as WONTFIX
